CrossMod/SortingOption.cs

```diff
@@ -40,17 +37,22 @@ public abstract partial class SortingOption : ModTexturedType, ILocalizedModType
 			ModTypeLookup<SortingOption>.Register(this);
 
 			Type = SortingOptionLoader.Add(this);
+
+			Tooltip = Language.GetOrRegister(Mod, $"SortingOption.{Name}");
```

Comment:
this code should not be here.  please use the existing code in MagicStorage/1.4.4 as a comparison